final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,146 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
-
+hero_name = 'Superman'
+special_ability = 'Flying'
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
-
+greeting = "Hello #{hero_name} how are you today?"
+catchphrase = "#{special_ability} faster than a speeding bullet"
 # Declare two variables - power AND energy - set to integers
-
+power = 50
+energy = 100
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
-
+full_power = power * 500
+full_energy = energy + 150
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
-
+is_human = true
+identity_concealed = true
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+arch_enemies = ['Lex Luthor', 'Brainiac', 'Bizarro']
+sidekicks = ['Superboy', 'Wonder Woman', 'Talon']
 # Print the first sidekick to your terminal
-
+p sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+p arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies[3] = 'Joker'
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+p arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.shift
 # Print the sidekicks array to terminal to ensure you added a new sidekick
-
+p sidekicks
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
 
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
 
+def assess_situation(danger_level, save_the_day, bad_excuse)
+
+  if danger_level > 50
+    p "#{bad_excuse}"
+
+  elsif danger_level > 10
+    p "#{save_the_day}"
+
+  else danger_level < 10
```

Comment:
The `else` part of an if/else conditional doesn't require an expression to evaluate, because this path happens if all other conditionals fail.  If you wanted to keep the conditional expression here (`danger_level < 10`) you would need to use `elsif` rather than `else`, but if that was the case - what would happen if if the `danger_level` passed in was 10?